Protobuild.Internal/BuildResources/GenerateProject.CSharp.xslt

```diff
@@ -938,7 +950,10 @@
           </xsl:otherwise>
         </xsl:choose>
         <xsl:choose>
-          <xsl:when test="$root/Input/Generation/Platform = 'Windows8' or $root/Input/Generation/Platform = 'WindowsPhone81'">
+			<xsl:when test="$root/Input/Generation/Platform = 'WindowsUAP'">
```

Comment:
Also don't mix this choose statement here; this choose statement is for the `<ProductVersion>` tag.  To introduce the `<FileAlignment>` tag, use a separate `<xsl:if>` or `<xsl:choose>` block.